packages/unified-selection/README.md

```diff
@@ -132,3 +132,35 @@ const selectionHiliteSet = await selectionHiliteProvider.getHiliteSet({ imodel.k
 // is shorter than that of `SelectionStorage`, to unregister the listener.
 selectionHiliteProvider.dispose();
 ```
+
+#### Selection scopes
+
+Selection scopes allow decoupling of what gets picked and what gets selected. Without selection scopes, whenever a user picks an element in the viewport, its ID goes straight into unified selection storage. With selection scopes we can modify that and add something different. The input to selection scopes' processor is a query executor, element IDs and the scope to apply, and the output an iterator of `SelectableInstanceKey`. We get the input when user picks some elements in the viewport, run that through selection scope processor and put the output into unified selection storage.
+
+Here are the scopes we support at the moment:
+
+- `element` - return key of selected element
+- `category` - return key of element's category
+- `model` - return key of element's model
+- `functional` - return key of element's related functional element
```

Comment:
```suggestion
- `element` - return key of selected element.
- `category` - return key of geometric element's category.
- `model` - return key of element's model.
- `functional` - return key of element's related functional element.
```

Also, `functional` scope's description sounds too simple compared to the implementation of how we find those related elements. I think we should include this information:
- what relationship is used to find the related functional element,
- what if there's no related functional element.

Reply:
Expanded on how the related functional element is found based on the type of element.

Reply:
Thanks, the explanation looks very helpful to me.